core/execution/future/liquidation.go

```diff
@@ -26,6 +26,9 @@ import (
 )
 
 func (m *Market) checkNetwork(ctx context.Context, now time.Time) error {
+	if m.as.InAuction() {
+		return nil
+	}
 	// debug
```

Comment:
not relevant but what is this?

Reply:
Not a clue, doesn't mean anything (anymore), so it's gone now